nucypher/blockchain/eth/sol/source/contracts/lib/UmbralDeserializer.sol

```diff
@@ -35,26 +35,45 @@ library UmbralDeserializer {
         CorrectnessProof proof;
     }
 
-    // TODO rename
     struct PreComputedData {
-        bytes data;
+        uint256 pointEyCoord;
+        uint256 pointEZxCoord;
+        uint256 pointEZyCoord;
+        uint256 pointE1yCoord;
+        uint256 pointE1HxCoord;
+        uint256 pointE1HyCoord;
+        uint256 pointE2yCoord;
+        uint256 pointVyCoord;
+        uint256 pointVZxCoord;
+        uint256 pointVZyCoord;
+        uint256 pointV1yCoord;
+        uint256 pointV1HxCoord;
+        uint256 pointV1HyCoord;
+        uint256 pointV2yCoord;
+        uint256 pointUZxCoord;
+        uint256 pointUZyCoord;
+        uint256 pointU1yCoord;
+        uint256 pointU1HxCoord;
+        uint256 pointU1HyCoord;
+        uint256 pointU2yCoord;
     }
 
-    uint8 constant BIG_NUM_SIZE = 32;
-    uint8 constant POINT_SIZE = 33;
-    uint8 constant SIGNATURE_SIZE = 64;
-    uint8 constant ORIGINAL_CAPSULE_SIZE = 2 * POINT_SIZE + BIG_NUM_SIZE;
-    uint8 constant CORRECTNESS_PROOF_SIZE = 4 * POINT_SIZE + BIG_NUM_SIZE + SIGNATURE_SIZE;
-    uint8 constant CAPSULE_FRAG_SIZE = 3 * POINT_SIZE + BIG_NUM_SIZE;
-    uint8 constant FULL_CAPSULE_FRAG_SIZE = CAPSULE_FRAG_SIZE + CORRECTNESS_PROOF_SIZE;
+    uint256 constant BIGNUM_SIZE = 32;
```

Comment:
Do these need to be of type `uint256`? Probably could be smaller, right?

Reply:
It's really doesn't matter, constant is not stored in contract storage

Reply:
Does it really not matter for gas when executed?

Reply:
No, because pointer is `uint256` and constant will be cast to `uint256`